doc/nrf/gs_programming.rst

```diff
@@ -212,7 +212,11 @@ After completing the :ref:`manual <build_environment_cli>` or :ref:`automatic <g
 
       .. note::
 
-	     To build from a directory other than the application directory, run the west build command with an additional parameter *directory_name*,  specifying the application directory.
+         To build from a directory other than the application directory, run the west build command with an additional parameter *directory_name*,  specifying the application directory.
+
+         To get extra device tree overlays with new features available for a board version, run the west build command with an additional parameter *@board_revision*.
+         The *board_revision* is printed on the label of your DK, just below the PCA number.
+         For example, if you run the west build command with an additional parameter *@0.14.0* for nRF9160 build target, it adds the external flash on the nRF9160 DK that was available since board version 0.14.0.
```

Comment:
Can we use a newer revision than `0.14.0`? Perhaps `1.0.0` or `1.0.1`? The `0.14.0` is an old board, and it also doesn't make it clear that the board revision doesn't have to be one of the defined versions in revision.cmake for the board (if anyone looks into that).